### DISCUSSION TITLE
Recommendation for stepper and images/third-party logos

### DISCUSSION THREAD

**Discussion by mindytom** [twilio-labs/paste#1122] [Q&A] [answered]
Hey y'all! 

I'm currently working on a wizard experience for Event Streams and see that there's no stepper component for Paste today. Based from a [previous post](https://github.com/twilio-labs/paste/discussions/623) from July '20, I'm thinking of using Katie's suggestion of using the box and text primitives. I'd like to know if this is still the most up to date recommendation? 

I'd also like to know if there's any recommendations for using images (either PNG or SVG) specifically third-party logos? 

Please view my latest [prototype](https://twilio.invisionapp.com/share/CPZXED8M5HG#/443283911_Step_1) to see how I'm thinking of using these. 

Thanks!
Mindy

**Comment by corinne-nagel** [comment, marked as answer]
Hey @mindytom , the stepper component is on the roadmap here: https://paste.twilio.design/roadmap and it's not planned for until the end of this year. So yep, the rec from Katie still stands. You may have already done this, but just to be sure, I would suggest that you share this with other designers as it may be something someone else has designed and would be great to be consistent on (Ishita may have done something similar?).

I would always suggest an svg if possible as the file sizes will be smaller. 

That design is looking great btw!

**Reply by mindytom** [reply]
Yeah that stepper is actually repurposed from Ishita's design. I just replaced the styling to match Paste. 

Thanks Corinne!
